x/blob/module.go

```diff
@@ -25,9 +24,9 @@ var (
 	_ module.HasGenesis          = AppModule{}
 	_ module.HasConsensusVersion = AppModule{}
 	_ module.HasName             = AppModule{}
+	_ module.HasServices         = AppModule{}
```

Comment:
@julienrbrt just double checking switching to `module.HasServices` won't break anything and is the correct behaviour here.